privacyidea/translations/de/LC_MESSAGES/messages.po

```diff
@@ -71,559 +71,623 @@ msgstr "Ungültiger Benutzer."
 msgstr "Eine eigene HTML-Vorlage für das Menü im Web UI verwenden."
 
-#: lib/policy.py:1756
+#: lib/policy.py:1991
 msgid "Use your own html template for the web UI baseline/footer."
 msgstr "Eine eigene HTML-Vorlage für die Fußzeile im Web UI verwenden."
 
-#: lib/policy.py:1760
+#: lib/policy.py:1995
 msgid "Whether the user ID and the resolver should be displayed in the token list."
 msgstr "Die Benutzer ID und den Benutzer-Resolver in der Tokenliste anzeigen."
 
-#: lib/policy.py:1765
+#: lib/policy.py:2000
 msgid "The URL of a repository, where the policy templates can be found.  (Default https: //raw.githubusercontent.com/ privacyidea/policy-templates /master/templates/)"
 msgstr "Die URL der Policy-Templates. (Default https: //raw.githubusercontent.com/ privacyidea/policy-templates /master/templates/)"
 
-#: lib/policy.py:1772
+#: lib/policy.py:2007
 msgid "As long as a user has no token, he will only see a token wizard in the UI."
 msgstr "Einem Benutzer ohne Token wird automatisch der Token Wizard angezeigt."
 
-#: lib/policy.py:1777
+#: lib/policy.py:2012
 msgid "The tokenwizard will be displayed in the token menu, even if the user already has a token."
 msgstr "Der Token Wizard als Menüpunkt angeboten - auch wenn der Benutzer bereits einen Token hat."
 
-#: lib/policy.py:1782
+#: lib/policy.py:2017
+msgid "The welcome dialog will be displayed if the user has no tokens assigned."
+msgstr "Wenn dem Benutzer kein Token zugewiesen ist, wird der Willkommensdialog angezeigt."
+
+#: lib/policy.py:2021
 msgid "This is the default token type in the token enrollment dialog."
 msgstr "Die Voreinstellung des Tokentyps im Rollout-Dialog."
 
-#: lib/policy.py:1788
+#: lib/policy.py:2027
 msgid "A list of realm names, which are displayed in a drop down menu in the WebUI login screen. Realms are separated by white spaces."
 msgstr "Eine durch Leerzeichen getrennte Liste von Realmnamen, die in einer Auswahlliste beim Login in das Webinterface angezeigt werden."
 
-#: lib/policy.py:1794
+#: lib/policy.py:2033
 msgid "If this checked, the administrator will not see the welcome dialog anymore."
 msgstr "Dem Administrator wird der Willkommen-Dialog nicht mehr angezeigt."
 
-#: lib/policy.py:1799
+#: lib/policy.py:2038
 msgid "Per default disabled actions result in disabled buttons. When checking this action, buttons of disabled actions are hidden."
 msgstr "Buttons von Funktionen, die ein Benutzer nicht ausführen darf, werden ausgeblendet. (Das Standardverhalten ist, dass die Buttons deaktiviert werden)"
 
-#: lib/policy.py:1804
+#: lib/policy.py:2043
 msgid "If this is checked, the seed will be displayed as text during enrollment."
 msgstr "Wenn dies angehakt ist, wird das Seed während des Rollouts als Text angezeigt."
 
+#: lib/policy.py:2099
+msgid "The policy only matches if certain conditions on the user info are fulfilled."
+msgstr "Die Richtlinie greift, wenn zusätzliche Bedingungen der Benutzerattribute erfüllt sind."
```

Comment:
maybe rather "wenn zusätzliche Anforderungen an die Benutzerattribute erfüllt sind"?

Reply:
done in 196c3585